scripts/Deploy.s.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script, stdJson} from "forge-std/Script.sol";
+import {L1ERC20Bridge} from "src/L1ERC20Bridge.sol";
+import {L2ERC20} from "src/L2ERC20.sol";
+import {Constants} from "test/Constants.sol";
+
+/// @notice Deploy L1 bridge and corresponding token to be minted on L2
+contract Deploy is Script, Constants {
+  using stdJson for string;
+
+  function run() public {
+    // Get the contract address for the L1 ERC20Votes token
+    string memory file = "broadcast/DeployFakeERC20.s.sol/43113/run-latest.json";
+    string memory json = vm.readFile(file);
+    address deployedL1Token = json.readAddress(".transactions[0].contractAddress");
+
+    setFallbackToDefaultRpcUrls(false);
+
+    vm.createSelectFork(getChain("polygon_mumbai").rpcUrl);
```

Comment:
similar question here—should the networks come from config?

Reply:
I think we could make chain id a config and get the correct rpcs. We would probably have an L1 chain id and an L2 chain id config.